test/features/t_links/sqlserver/sqlserver_t_links_period_mat.feature

```diff
@@ -15,7 +15,6 @@ Feature: [SQLS-TLK-PM] Transactional Links using Period Materialization
     And I stage the STG_CUSTOMER data
     And I insert by period into the T_LINK t_link by day
     And I insert by period into the T_LINK t_link by day
-    When I load the T_LINK t_link
```

Comment:
Why was this line

Reply:
With dbt-sqlserver this was causing the test scenario to fail. The equivalent Snowflake test scenario does not have this and that test scenario passes.

Reply:
This was causing the test scenario to fail. I don't know why it was there given that this is a period materialisation load.

Reply:
I missed that this was SQLServer, no worries